CHANGELOG.md

```diff
@@ -1,15 +1,19 @@
 ## HEAD (Unreleased)
-(None)
+
+- [sdk/dotnet] Fix creation of CustomResources (https://github.com/pulumi/pulumi-kubernetes/pull/1741)
 
 ## 3.7.2 (September 17, 2021)
+
```

Comment:
Let's revert the formatting changes to the changelog for this PR.

Reply:
Not sure where all of those formatting changes came from. I certainly didn't make them. That said, I'll patch it up.

Reply:
Ok. Changes reverted.